src/styles/components/_navbar.scss

```
@@ -48,16 +48,48 @@
     text-decoration: none;
 
     &--sign-in {
-      // TODO: for future implementation
+      color: $dark;
+
+      @include respond-below('sm') {
+        display: none;
+      }
+
+      @include respond-below('md') {
+        font-size: rem(12px);
+      }
```

Since the button must me hidden when smaller than `sm`, can we do this? Mobile first remember 😬
```suggestion
      display: none;
      color: $dark;

      @include respond-up('sm') {
        display: block;
        font-size: rem(12px);
      }

      @include respond-up('md') {
        font-size: rem(16px);
      }
```